src/core/lv2/Lv2Ports.cpp

```diff
@@ -168,6 +171,7 @@ std::vector<PluginIssue> Meta::get(const LilvPlugin *plugin,
 		issue(badPortType, "cvPort");
 		m_type = Type::Cv;
 	}
+	else if (isA(LV2_ATOM__AtomPort)) { m_type = Type::AtomSeq; }
```

Comment:
Should this first check that the atom port supports MIDI, and add an "unknown port type" issue if not?

Reply:
I talked with the lv2-devs in the lv2 IRC channel, and they said it's OK to have atom sequence ports, even if you don't support any of its types. A malfunction is not expected in this case. I added a justification comment into the code though.

One other issue I found here is that the line you marked assumes that every atom port is an atom sequence port. I fixed this in the same commit (7e059b3).

Reply:
I'll mark this as resolved, feel free to re-open.